ffi/framebuffer_pocketbook.lua

```diff
@@ -16,10 +16,10 @@ local function _getPhysicalRect(fb, x, y, w, h)
 end
 
 local function _adjustAreaColours(fb)
-    if fb.device.hasColorScreen and fb.device.color_saturation then
-        fb.debug("adjusting image color saturation", fb.device.color_saturation)
```

Comment:
You might also want to delete the color_saturation settings from all the pocketbook device in:

https://github.com/koreader/koreader/blob/ee7c3ab551f320ff0095c276bdadfac77beeee69/frontend/device/pocketbook/device.lua

Reply:
Of course, it's in the separate PR: https://github.com/koreader/koreader/pull/11597